examples/README.md

```diff
@@ -0,0 +1,101 @@
+- `<category>` - category of the code examples. The main category is `docs` which means: code examples from the documentation.
+- `<framework>` - any supported framework's name (react, angular, vue, js).
+- `<example_path>` - path to the specific code example. A folder name of the code example will identify that code example - it will be used as a URL further. The folder name of the code example should be the same across all the frameworks.
+
+An example path to the code example: `examples/8.1.0/docs/angular/custom-id`
+
+It's worth noting, that the `examples` directory is defined as a `npm workspace`, as well as each of the `<framework>` directories within it. This allows installing shared dependencies for all the framework-specific examples.
+
+### Live on production
+
+All code examples are available online. The base URL for the code examples is https://examples.handsontable.com/handsontable/ and after the slash comes the path to the built project. 
+
+URL to the specific project follow the undermentioned convention:
+
+`<version_number>/<category>/<framework>/<example_path>`
+
+An example URL to the live example: https://examples.handsontable.com/handsontable/8.1.0/docs/js/settings
+
+For more details see the [Folder structure](#folder-structure) section.
+
+### Preview on CodeSandbox
+
+To preview the arbitary code example on CodeSandbox you must get the link to the project folder in the Gihub repo. You will find all code examples in our repo https://github.com/handsontable/handsontable/tree/develop/examples.
+
+For example: if the URL to the project is: https://github.com/handsontable/handsontable/tree/develop/examples/8.0.0/docs/js/settings you can preview that project on CodeSandbox adding the "box" word right after the "github" and before the ".com". The URL to the sandboxed project would be https://githubbox.com/handsontable/handsontable/tree/develop/examples/8.0.0/docs/js/settings.
+
+### Creating new examples
+
+Read the [Templates Guideline](./templates/README.md) for the detailed information about new code examples creation.
+
+### Editing existing examples
+
+Sometimes you want to edit existing code examples that is live in the `/examples/<version_number>` directory.
+
+1. Edit code examples within the `/examples/<version_number>` directory.
+2. Commit and push the code examples to the repo.
+3. In Github Actions run a manual workflow with the `<version_number>` input of the Handsontable and Handsontable's wrapper version.
+4. The code examples will be deployed to the Github Pages and will be available under the same URL as they already were.
+
+### Development
+
+To see code examples in action run these commands:
+
+1. `npm examples:install <version_number>` - will install the dependencies of all the examples matching the `<version_number>`, utilizing the `examples`' internal workspace logic.
```

Comment:
@Osky772 you forgot to add the word `run` here.

Reply:
I think that we can add a note before the commands, that the examples must be created already

Reply:
IMO, the instructions should be clear enough that you can paste them into the console without unnecessary analysis.
Here you have to enter before `npm run examples: version <version_number>` which is not consistent with the next instructions, because you enter `npm run examples: install 8.3.2` without the word **version**. It is not consistent. As for me, it confuses me.

Reply:
This is about "development" of already created versioned examples. `npm run examples:version` is a part of a different process. Running it here is totally unnecessary most of the time.

`npm run examples:version` is a command for the release master. Most of us will/should work with `next` dir